docs/guides/web-servers/caddy/compile-caddy-from-source/index.md

```diff
@@ -19,58 +19,46 @@ aliases: ['/web-servers/caddy/compile-caddy-from-source/']
 
 [Caddy](https://caddyserver.com/) is a fast, open-source and security-focused web server written in [Go](https://golang.org/). Caddy includes modern features such as support for virtual hosts, minification of static files, and HTTP/2. Caddy is also the first web-server that can obtain and renew SSL/TLS certificates automatically using [Let's Encrypt](https://letsencrypt.org/).
 
-Caddy has recently updated their license, clearly defining what is considered personal or enterprise use. A commercial license is now required for commercial or enterprise use, including any installation that uses precompiled Caddy binaries. However, because the project is Apache licensed, by building it from source you have access to the original, [Apache-licensed web server](https://twitter.com/mholt6/status/908041929438371840).
+## Before you Begin
 
-## Build Caddy from Source
 ### Install Go
 
-1. You will need a current version of Go installed on your Linode. Complete the steps in our guide on [installing Go](/docs/development/go/install-go-on-ubuntu/).
+1. You need the latest version of Go installed on your Linode. Complete the steps in our guide on [installing Go](/docs/development/go/install-go-on-ubuntu/).
 
-1. Print your Go installation's current `$GOPATH`:
+### Install xcaddy
 
-        go env GOPATH
+Install the latest version of `xcaddy`, a command line tool that downloads and builds Caddy and its plugins for you easily.
 
-1. Set the transitional environment variable for Go modules.
+1. Download and install `xcaddy`:
 
-        cd $GOPATH/src
-        export GO111MODULE=on
+        sudo apt install -y debian-keyring debian-archive-keyring apt-transport-https
+        curl -1sLf 'https://dl.cloudsmith.io/public/caddy/xcaddy/gpg.key' | sudo apt-key add -
```

Comment:
`apt-key` is deprecated, the `xcaddy` docs were just recently changed to fix it:

```suggestion
        curl -1sLf 'https://dl.cloudsmith.io/public/caddy/xcaddy/gpg.key' | sudo tee /etc/apt/trusted.gpg.d/caddy-xcaddy.asc
```